model/converter/thrift/jaeger/from_domain_test.go

```diff
@@ -0,0 +1,147 @@
+package jaeger
```

Comment:
the script only adds headers to files that are in git, so you should `git add` them first, then run `make fmt`. But the build should fail anyway if you don't have a header.